service/dap/server_test.go

```diff
@@ -152,31 +274,54 @@ func TestSetBreakpoint(t *testing.T) {
 		client.ExpectSetExceptionBreakpointsResponse(t)
 
 		client.ConfigurationDoneRequest()
-		cdResp := client.ExpectConfigurationDoneResponse(t)
-		if cdResp.RequestSeq != 4 {
-			t.Errorf("got %#v, want RequestSeq=4", cdResp)
-		}
+		client.ExpectConfigurationDoneResponse(t)
+		// This triggers "continue"
+
+		// TODO(polina): add a no-op threads request
+		// with dummy response here once server becomes async
+		// to match what happens in VS Code.
 
-		client.ContinueRequest(1)
 		stopEvent1 := client.ExpectStoppedEvent(t)
 		if stopEvent1.Body.Reason != "breakpoint" ||
 			stopEvent1.Body.ThreadId != 1 ||
 			!stopEvent1.Body.AllThreadsStopped {
 			t.Errorf("got %#v, want Body={Reason=\"breakpoint\", ThreadId=1, AllThreadsStopped=true}", stopEvent1)
 		}
-		client.ExpectContinueResponse(t)
+
+		client.ThreadsRequest()
+		tResp := client.ExpectThreadsResponse(t)
+		if len(tResp.Body.Threads) != 5 { // 1 main + 4 runtime
+			t.Errorf("\ngot  %#v\nwant len(Threads)=5", tResp.Body.Threads)
+		}
+		var gotThread = func(want dap.Thread) (got bool) {
+			for _, t := range tResp.Body.Threads {
+				got = got || reflect.DeepEqual(t, want)
+			}
+			return got
+		}
+		// TODO(polina): can we reliably test for these values?
```

Comment:
I see other tests in delve refer to the `Increment` goroutine, but not `runtime.gopark`.

To make it a bit more resilient, you may want to do an approximate check on the name - i.e. that it starts with `runtime`.